src/components/SummaryKeyComponents/AbstractAccordionItem.js

```diff
@@ -0,0 +1,42 @@
+import { LOADING_TEXT } from "../../Constants";
+
+/**
+ * Class representing the Abstract Accordion Item in DOM
+ * @param {HTMLElement} components.abstractText the text container in the accordion
+ */
+class AbstractAccordionItem {
+  components = {
+    abstractText: document.querySelector("#abstract-text"),
+  };
+
+  /**
+   * Set the text of the accordion item
+   * @param text text to be set
+   */
+  setText = (text) => {
```

Comment:
setText, setError, setLoading and clear repeat themselves 4 times through these 4 classes.